tuxemon/monster.py

```diff
@@ -549,6 +552,27 @@ def set_moves(self, level: int) -> None:
                 tech.load(ele)
                 self.learn(tech)
 
+    def update_moves(self, levels: int) -> Optional[Technique]:
+        """
+        Set monster moves according to the levels increased.
+        Excludes the moves already learned.
+
+        Parameters:
+            level: Nr of levels.
+
+        """
+        tech = None
```

Comment:
Something is wrong here.

Why do we need this extra `tech` variable here? Why not just return `technique` which relates to the same thing.

Also, according to the method logic, multiple techniques can be learnt (because we are looping over a list)... So why are we returning only one? Do we just assume it's always going to be one (in this case, we could stop earlier the call by putting a return statement inside of the if block), or what?

Reply:
My mistake, you're right.